File: taller/AMarotoChivite-taller/AMarotoChivite-taller-Java/src/main/java/simulacionTaller/factories/MultiFactory.java
package simulacionTaller.factories;


import simulacionTaller.models.Multi;
import simulacionTaller.models.Worker;

import static simulacionTaller.utils.randomInfo.*;

public class MultiFactory {
    public static Multi create() {
        String name = randomName();
        int experienceYear = randomAnno();
        int salary = randomSalary() + 350;
        int dailyHours = randomDailyHours();

        return new Multi(name, experienceYear, salary, dailyHours, Worker.BossAssigment.WITHOUT_BOSS);
    }
}

File: taller/AMarotoChivite-taller/AMarotoChivite-taller-kotlin/src/main/kotlin/simulacionTaller/factories/MultiFactory.kt
package simulacionTaller.factories

import simulacionTaller.models.Multi
import simulacionTaller.models.Worker.BossAssigment
import simulacionTaller.utils.randomAnno
import simulacionTaller.utils.randomDailyHours
import simulacionTaller.utils.randomName
import simulacionTaller.utils.randomSalary

object MultiFactory {
    @JvmStatic
    fun create(): Multi {
        val name = randomName()
        val experienceYear = randomAnno()
        val salary = randomSalary() + 350
        val dailyHours = randomDailyHours()
        return Multi(name, experienceYear, salary.toDouble(), dailyHours, BossAssigment.WITHOUT_BOSS)
    }
}